web/webkit/src/main/scala/net/liftweb/http/Templates.scala

```diff
@@ -28,7 +28,10 @@ import java.io.InputStream
  * Contains functions for obtaining templates
  */
 object Templates {
-  private val suffixes = LiftRules.templateSuffixes
+  // Making this lazy to ensure it doesn't accidentally init before Boot completes in case someone touches this class.
+  private lazy val parsers = LiftRules.contentParsers.flatMap(parser =>
```

Comment:
I'd prefer it if we were explicit about this and named it `parserForSuffix`. Though this brings up the question: what happens if two parsers sign up for the same suffix? Right now, we'll just lose the other one, and the order is reversed. This seems somewhat problematic… At minimum, I feel like we should:
 - Be prescriptive about what order they'll be considered in (i.e., in the `LiftRules` docs, say “if multiple things register for the same suffix, the (first|last) one will win”).
 - Think about alternate approaches to first/last one wins. Don't have any suggestions right now, and I don't think we need that solution before merging this stuff, but I think it's worth sticking it in the back of our minds and seeing if we can think of something.